### DISCUSSION TITLE
是否会收录一些自动化插件的功能，以及一些直播界面增强功能

### DISCUSSION THREAD

**Discussion by 8MiYile** [the1812/Bilibili-Evolved#4488] [功能建议 / Ideas]
https://greasyfork.org/zh-CN/scripts/406048-bilibili-live-tasks-helper
https://greasyfork.org/zh-CN/scripts/432743-b%E7%AB%99%E7%9B%B4%E6%92%AD%E5%BE%BD%E7%AB%A0%E5%88%87%E6%8D%A2%E5%A2%9E%E5%BC%BA
https://greasyfork.org/zh-CN/scripts/432741-b%E7%AB%99%E7%9B%B4%E6%92%AD%E8%81%8A%E5%A4%A9%E5%AE%A4%E5%BC%B9%E5%B9%95%E5%8F%91%E9%80%81%E5%A2%9E%E5%BC%BA
就是 会不会合并这些实用的功能

**Comment by Geeyun-JY3** [comment]
兼容的话直接用不就完了……为啥要合并

而且已经明确过 BLHT 不会整合到 Bilibili Evolved：https://github.com/the1812/Bilibili-Evolved#blth---bilibili-live-tasks-helper
